mobile/lib/modules/home/ui/asset_grid/immich_asset_grid_view.dart

```diff
@@ -254,9 +264,8 @@ class ImmichAssetGridViewState extends State<ImmichAssetGridView> {
             widget.margin * (widget.assetsPerRow - 1) / widget.assetsPerRow;
         final rows =
             (section.count + widget.assetsPerRow - 1) ~/ widget.assetsPerRow;
-        final List<Asset> assetsToRender = scrolling
-            ? []
-            : widget.renderList.loadAssets(section.offset, section.count);
+        final List<Asset> assetsToRender =
```

Comment:
This loads many assets quickly and often from DB while scrolling. We'd need to test the performance implications for large libraries (500k assets) on medium-powered devices.

Reply:
Yes. This does load the assets for each of the section during scrolling. I gave it a test only in the simulator using the demo instance which is not much. This change is to also show the thumb hash on scrolling with the scroll bar.

The main change that vastly reduced the flickering was using the `RepaintBoundary` around `ThumbnailImage`. I tested it after removing all the changes I made from `immich_asset_grid_view` and except wrapping the thumbnails with repaintboundary.

We can remove the thumbhash on scroll change if the performance is bad on low - medium end devices. I don't have that many assets to test 😅  Would be great if someone could test it